.github/workflows/review-release-notes.yml

```diff
@@ -12,22 +16,30 @@ jobs:
           fetch-depth: 2
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v18.7
-        with:
-          separator: ';'
-      - name: Setup Python
-        uses: actions/setup-python@v3
+        uses: tj-actions/changed-files@v35.1.0
         with:
-          python-version: '3.9'
-      - name: Setup Poetry
-        uses: Gr1N/setup-poetry@v7
-      - name: Install Python Dependencies
-        run: |
-          poetry install --with ci
+          separator: ${{ env.CHANGED_FILES_DELIMITER }}
+          files: |
+            Packs/**/ReleaseNotes/**
+          since_last_remote_commit: false
+
+      - name: Install SDK
+        run: python3 -m pip install demisto-sdk
+
       - name: Review Release Notes
+        if: steps.changed-files.outputs.all_changed_files != '' && steps.changed-files.outputs.all_changed_files != ' '
         run: |
-          changed_files="${{ steps.changed-files.outputs.all_changed_files }}"
-          delim=';'
-          echo "Starting docs review"
-          poetry run python Utils/github_workflow_scripts/run_docs_review.py --changed_files "$changed_files" --delimiter $delim
-          echo "Finished docs review successfully"
\ No newline at end of file
+          release_notes="${{ steps.changed-files.outputs.all_changed_files }}"
+          echo "Release notes found in PR: ${release_notes}"
+          release_notes_to_review=$(Utils/github_workflow_scripts/get_xsoar_supported_release_notes.py "$release_notes" --delimiter ${{ env.CHANGED_FILES_DELIMITER }})
+
+          # Check if there are any release notes
+          if [ $? -eq 0 ] && [ -n "$release_notes_to_review" ]; then
```

Comment:
The script should work when passing empty arguments, so I feel like this is redundant here and could potentially hide deeper problems with the sdk command.

Reply:
Relevant to discussion https://github.com/demisto/content/pull/23278#discussion_r1059743911.